### ISSUE TITLE
Add isort to the Travis build

### ISSUE THREAD

**Issue by kelseyq** [SexualHealthInnovations/callisto-core#59]
related to #41 

ideas on how to do this: https://bugzilla.mozilla.org/show_bug.cgi?id=1192957

**Comment by kevindaum** [comment]
I'll have to dig into what Mozilla did, but the problem we're going to have with this right now is related to the [feature request I filed with isort](https://github.com/timothycrosley/isort/issues/443). Since isort puts the django imports at the very bottom, I had to manually move them all up after running isort. So if I run `isort --check-only -rc .` right now, I get a bunch of errors like `ERROR: /Users/kevin/projects/other/callisto-core/callisto/delivery/admin.py Imports are incorrectly sorted.`. I'm inclined to wait to see if someone who knows isort better than me volunteers to work on that issue before I spend time on that just to accomplish this.

**Comment by kevindaum** [comment]
I wonder if and how the django team automatically enforces sort order within django.

**Comment by kelseyq** [comment]
It looks like there's a flake8 plugin that can check it: https://github.com/PyCQA/flake8-import-order

This reveals my biases coming from a statically typed world, but I'm strongly against adding style guide requirements that can't be checked by machine. I will try to keep up on import order in PRs as we go, but I'm going to wait on merging #58 until we've found an automated check of some kind. Let's revisit after the PEP8 checks have been added.
